Wikimate.php

```diff
@@ -1,41 +1,107 @@
 <?php
-/// =============================================================================
-/// Wikimate is a wrapper for the MediaWiki API that aims to be very easy to use.
-///
-/// @version    0.13.0
-/// @copyright  SPDX-License-Identifier: MIT
-/// =============================================================================
+/**
+ * =============================================================================
+ * Wikimate is a wrapper for the MediaWiki API that aims to be very easy to use.
+ *
+ * @package    Wikimate
+ * @version    0.13.0
+ * @copyright  SPDX-License-Identifier: MIT
+ * =============================================================================
+ */
 
 /**
  * Provides an interface over wiki API objects such as pages and files.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
  * @since   December 2010
  */
 class Wikimate
 {
 	/**
-	 * @var  string  The current version number (conforms to http://semver.org/).
+	 * The current version number (conforms to Semantic Versioning).
+	 *
+	 * @var string
+	 * @link https://semver.org/
 	 */
 	const VERSION = '0.13.0';
 
+	/**
+	 * CSRF token
+	 *
+	 * @var string
+	 * @link https://www.mediawiki.org/wiki/Special:MyLanguage/API:Tokens
+	 */
 	const TOKEN_DEFAULT = 'csrf';
-	const TOKEN_LOGIN   = 'login';
 
+	/**
+	 * Login token
+	 *
+	 * @var string
+	 * @link https://www.mediawiki.org/wiki/Special:MyLanguage/API:Tokens
+	 */
+	const TOKEN_LOGIN = 'login';
+
+	/**
+	 * Base URL for API requests
+	 *
+	 * @var string
+	 * @link https://www.mediawiki.org/wiki/Special:MyLanguage/API:Main_page#Endpoint
+	 */
 	protected $api;
+
+	/**
+	 * Username for API requests
+	 *
+	 * @var string
+	 * @link https://www.mediawiki.org/wiki/Special:MyLanguage/API:Login#Method_1._login
+	 */
 	protected $username;
+
+	/**
+	 * Password for API requests
+	 *
+	 * @var string
+	 * @link https://www.mediawiki.org/wiki/Special:MyLanguage/API:Login#Method_1._login
+	 */
 	protected $password;
 
-	/** @var  Requests_Session */
+	/**
+	 * Session object for HTTP requests
+	 *
+	 * @var Requests_Session
+	 * @link https://requests.ryanmccue.info/
+	 */
 	protected $session;
+
+	/**
+	 * User agent string for Requests_Session
+	 *
+	 * @var string
+	 * @link https://requests.ryanmccue.info/docs/usage-advanced.html#session-handling
+	 */
 	protected $useragent;
 
-	protected $error     = null;
+	/**
+	 * Error array with API and Wikimate errors
+	 *
+	 * @var array|null
+	 */
+	protected $error = null;
+
+	/**
+	 * Should we output debug logging?
```

Comment:
This wording could be confusing and seem like a TODO/FIXME note. How about this?
```suggestion
	 * Whether to output debug logging
```

Reply:
Saw the 'should?' wording in Requests and elsewhere a few times, but 'whether' is fine too.